CONTRIBUTING.md

```diff
@@ -1,45 +1,21 @@
 # Contributing to ActiveShipping
 
-We welcome fixes and additions to this project. Fork this project, make your changes and submit a pull request!
+We love receiving pull requests! Anything from new carriers to documentation fixes.
 
-### Code style
+## Standards
 
-Please use clean, concise code that follows Ruby community standards. For example:
+* PR should explain what the feature does, and why the change exists.
+* PR should include any carrier specific documentation explaining how it works.
+* Code _must_ be tested, including both unit and remote tests where applicable.
+* Be consistent. Write clean code that follows Ruby community standards.
```

Comment:
Should we link example documentation? https://github.com/bbatsov/ruby-style-guide is a great public resource that most people know of but having a reference is pretty good.